.changeset/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
```

Comment:
@jmrossy this is kind of a clever way to get versioned schema support in editor without local relative paths